mlos_core/mlos_core/optimizers/optimizer.py

```diff
@@ -254,7 +254,7 @@ def _from_1hot(self, config: npt.NDArray) -> pd.DataFrame:
                     j += 1
         return pd.DataFrame(df_dict)
 
-    def _to_1hot(self, config: Union[pd.DataFrame, pd.Series]) -> npt.NDArray:
+    def _to_1hot(self, *, config: Union[pd.DataFrame, pd.Series]) -> npt.NDArray:
```

Comment:
```suggestion
    def _to_1hot(self, config: Union[pd.DataFrame, pd.Series]) -> npt.NDArray:
```

Same here, though again, I'm not against leaving it.